app/views/dashboard/all_posts.html.erb

```diff
@@ -0,0 +1 @@
+<h1>All Posts Template</h1>
```

Comment:
Just a heads up, you can start taking care of internationalization  from beginning :)

Reply:
This will be great not to miss anything. It is possible to do it at the end, but perhaps a better idea to do it as you go, as Sagarpreet suggests!

Reply:
I was planning on creating some FTO issues for translations as well. I'll see how I can implement it partially and create some FTOs if possible.